WIP 3.0 Classes/Priest.md

```diff
@@ -509,717 +568,122 @@ You learn the *calm emotions* spell and always have it prepared. In addition, yo
 
 ##### Shadow Calling Spells
 Priest Level| Spells
 ------------|-----------
-1st |*cause fear*, *✦ dark void*
-3rd |*mind flay*, *phantasmal force*
+1st |*cause fear* ^XGE^, *✦ dark void*
+3rd |*✦ mind flay*, *phantasmal force*
 5th |*✦ void shift*, *vampiric touch*
 7th |*evard's black tentacles* *✦ schism*
-9th |*enervation ^XGE^*, *contagion*
+9th |*contagion*, *enervation* ^XGE^
 
 #### Psychic Voice 
-*1st-level priest feature*<br/>
+*1st-level Shadow feature*<br/>
 <div style='margin-top:-4px;'></div>
 
 You can form a telepathic connection between creatures of your choice you can see within 30 feet of you. Choose a number of creature you can see (including yourself), up to your proficiency bonus. The chosen creatures can speak telepathically with each other as long as they are on the same plane. To understand each other, the creatures must speak mentally in a language the other creature knows. 
 
-The telepathic connection last for a number of minutes equal to your priest level, and once used, cannot be used again until you finish a short or long rest, or unless you expend a spell slot of 1st level or higher to use it again. 
-
-\pagebreakNum
+The telepathic connection last for a number of minutes equal to your priest level, and once used, cannot be used again until you finish a short or long rest, unless you expend a spell slot of 1st level or higher to use it again. 
 
 #### Mind Sear 
-*1st-level priest feature*<br/>
+*1st-level Shadow feature*<br/>
 <div style='margin-top:-4px;'></div>
 
-Once on one of your turns as a bonus action, you can choose one creature affected by one of your priest spells or abilities. That creature, and all creatures of your choice within 5 feet of them must succeed on a Intelligence saving throw or take psychic damage equal to half your priest level. Any charmed or frightened creatures who take this damage takes an additional 1d6 psychic damage.
+As a bonus action, you can choose one creature affected by one of your priest spells or abilities. That creature, and all creatures of your choice within 5 feet of them, must succeed on a Intelligence saving throw or take psychic damage equal to half your priest level. Any charmed or frightened creatures who take this damage takes an additional 1d6 psychic damage.
 
 You can use this bonus action a number of times equal to your proficiency bonus, and regain all expended uses when you finish a long rest. 
 
 #### Devotion: Shadowform 
-*2nd-level priest feature*<br/>
+*2nd-level Shadow feature*<br/>
 <div style='margin-top:-4px;'></div>
 
-You can expend one use of Devotion and enshroud your body in shadow energy as a bonus action. For one minute, you gain the following benefits: 
+You can expend one use of Devotion and enshroud your body in shadow energy as a bonus action. For one minute, you gain the following effects:
 * You can't cast spells that restore hit points, unless it is one of your Shadow Calling spells. 
```

Comment:
Three spells actually! Enervation, void shift (in a way since it moves hit points, so might be worth updating the wording on the spell to make it clear) and Vampiric touch. Again hence the wording over calling out just Vampiric touch